src/Passes/Source/ValidationPass/ValidationPassConfiguration.hpp

```diff
@@ -0,0 +1,93 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Commandline/ConfigurationManager.hpp"
+#include "Types/Types.hpp"
+
+namespace microsoft
+{
+namespace quantum
+{
+
+    class ValidationPassConfiguration
+    {
+      public:
+        using Set = std::unordered_set<std::string>;
+        // Setup and construction
+        //
+
+        /// Setup function that attached the configuration to the ConfigurationManager.
```

Comment:
Don't see how this function can attach something. It just calls a few functions on config passing strings and a bool. No connection or attachment are created between objects. Maybe a better comment is needed.

Reply:
I have updated the text slightly. I think the core issue is that the documentation one needs to read to understand this, is that of ConfigurationManager. The new comment reflects this.

The configuration manager creates a reference to the boolean parameter passed and stores this for whenever it encounters the corresponding text flag. It then parses the value and updates the variable held by the class instance through the passes reference.